cmd/app/main.go

```diff
@@ -46,14 +51,45 @@ func main() {
 	var accRepoDb = postgres.NewAccountRepoDB(userRoleRepoDB, db)
 	var accService = services.NewAccountService(accRepoDb, accRepoDb, accRepoDb)
 
+	var scooterRepo = postgres.NewScooterRepoDB(db)
+	var grpcScooterService = services.NewGrpcScooterService(scooterRepo)
+	var scooterService = services.NewScooterService(scooterRepo)
+
+
 	sessStore := sessions.NewCookieStore([]byte(sessionKey))
 	authService := services.NewAuthService(userRoleRepoDB, sessStore)
 
 	handler := routing.NewRouter(authService)
 	routing.AddUserHandler(handler, userService)
 	routing.AddAccountHandler(handler, accService)
+	routing.AddScooterHandler(handler, scooterService)
+	routing.AddGrpcScooterHandler(handler, grpcScooterService)
 	httpServer := httpserver.New(handler, httpserver.Port(configs.HTTP_PORT))
 
+
+	svr := grpcserver.NewServer()
+	svr.Run()
+	grpcServer := grpc.NewServer()
+
+	protos.RegisterScooterServiceServer(grpcServer, svr)
+
+	listener, err := net.Listen("tcp", ":8000")
+	if err != nil {
+		panic(err)
+	}
+	go func() {
+		fmt.Println("grpc server started: 8000")
+		log.Fatal(grpcServer.Serve(listener))
+	}()
+
+	http.HandleFunc("/scooter", svr.ScooterHandler)
+	http.HandleFunc("/run", routing.StartScooterTrip)
+	http.HandleFunc("/", routing.ShowTripPage)
+
+
+	fmt.Println("http server started: 9000")
+	http.ListenAndServe(":9000", nil)
+
```

Comment:
I think you should try to put this block  (p. 70-91) in routing/handler.go file and refactor it, so we can use 1 HTTP server (the one we already have there)